packages/recoil/recoil_values/Recoil_atom.js

```diff
@@ -135,6 +135,7 @@ export type AtomEffect<T> = ({
   ) => void,
 
   // Accessors to read other atoms/selectors
+  observe: <S>(RecoilValue<S>, (value: S) => void) => () => void,
```

Comment:
Type definitions and test would also have to be added in `/typescript`

Reply:
Aye aye